docs/src/gs/create-workload-cluster.md

```diff
@@ -98,6 +98,55 @@ clusterctl generate cluster <cluster-name>\
 --from cluster-template-oraclelinux.yaml | kubectl apply -f -
 ```
 
+## Create a workload cluster in an alternative region
+
+CAPOCI provides a way to launch and manage your workload cluster in multiple
+regions. Choose the `cluster-template-alternative-region.yaml` template when
+creating your workload clusters from the [latest released artifacts][latest-release].
+Currently, the other templates do not support the ability to change the workload
+cluster region. 
+
+Each cluster can be further configured with the parameters
+defined in [Workload Cluster Parameters](#workload-cluster-parameters) and
+additionally with the parameter below.
+
+| Parameter             | Default Value                                                                | Description                                                                                                                 |
+|-----------------------|------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------|
+| `OCI_WORKLOAD_REGION` | `OCI_REGION` value from [Configure authentication][configure-authentication] | The [region](https://docs.oracle.com/en-us/iaas/Content/General/Concepts/regions.htm) to launch the workload cluster in.    |
+
+Below is an example of configuring the CAPOCI provider to authenticate in
```

Comment:
"The following example configures the CAPOCI provider to authenticate in ... and launch a workload cluster in ...."